extensions/amp-google-document-embed/validator-amp-google-document-embed.protoascii

```diff
@@ -37,6 +37,8 @@ tags: {  # <amp-google-document-embed>
     }
     blacklisted_value_regex: "__amp_source_origin"
   }
+  attrs: { name: "[src]" }
```

Comment:
nit: we usually have a comment before amp-bind attributes with `# amp-bind`. Not necessary though.